docs/contributing/releasing-islandora.md

```diff
@@ -44,7 +73,7 @@ ossrhPassword = jirapass
 signing.keyId = pubkeyid
 signing.password = keypassphrase
 signing.secretKeyRingFile = /your/home/.gnupg/secring.gpg 
-```
+```J
```

Comment:
Sorry, was this **J** character intentional?

Reply:
oops